doc/components/parallel_prefix_operations.md

```diff
@@ -0,0 +1,12 @@
+# Parallel Prefix Operations
+
+ROHD HCL implements a set of parallel prefix compute operations using different parallel prefix computation trees based on the ['ParallelPrefix'] node class carrying carry/save or generate/propagate bits.
+
+For example, we have unary operations like a word-level 'or' [`ParallelPrefixOrScan`] class, and a priority encoder [`ParallelPrefixPriorityEncoder`] class which computes the position of the first bit set to '1'. We have simple unary arithmetic operations like an increment [`ParallelPrefixIncr`] class, and a decrement [`ParallelPrefixDecr`] class. Finally, we have a binary adder [`ParallelPrefixAdder`] class. For background on basic parallel prefix adder structures, see <https://en.wikipedia.org/wiki/Kogge%E2%80%93Stone_adder>.
+
+Each of these operations can be implemented with different ['ParallelPrefix'] types:
+
+- ['Ripple']
```

Comment:
oh -- these were pointers to the html which I stripped out because of the markdown linting problem (missing links).
My intent is to put the links back, but we have a chicken-and-egg problem of pointing to files that have yet to be built.

Reply:
Sure, sounds good